setup/terminal.md

```diff
@@ -1,33 +1,49 @@
-* doing things on terminal force you to understand how things work on the lower level, quite often GUIs are using a subset of command line options. When you know how to use things from the command line you can do more complicated operations which are not supported from GUI level
-* It looks cool when your friends are looking over your shoulder ;-)
+* common operations on files are much faster when you do them from the terminal, as quite often typing a few commands in your terminal can do as much as clicking a thousand times on your screen)
+* doing things on terminal forces you to understand how things work on the lower level, quite often GUI is only a frontend for a more powerful command line application, and is exposing only a subset of all available command line options. When you know how to use things from the command line you can do more complicated operations, which are not supported from the GUI level.
+* It also looks cool when your friends are looking over your shoulder ;-)
 
 ### Speed
 
-Let's use simple example to demonstrate the difference in speed 
+Let's use a simple example to demonstrate the difference in speed 
 Imagen you have a folder in which you keep your notes, maybe 10 files with just few lines of text in each. Now you remember that in one of the notes you wrote something about your friend Alice and now you want to find this file out
 
-Below a comparison how one might do it using GUI and terminal 
-On the left user clicks through every file and look at the preview to see if there is anything about Alice (here we assume the notes are short and the full content would fit into the preview box)
-On the right user is doing the same opertion using the terminal and the command `grep`  
+Below is a comparison using GUI and terminal.
+On the left, the user clicks through every file and looks at the preview to see if there is anything about Alice, (assuming the notes are short and the full content would fit into the preview box)
+On the right the user is doing the same opertion using the terminal and the command `grep`  
 
 ![](img/alice.gif)
 
-You see that was only 5 files, now imagen if you have to search through a hundred files or a thousand files. The sample directory with notes files can be found [here](terminal-e)
+That was only 5 files, now imagen if you have to search through a hundred or a thousand files. The sample directory with the notes files can be found [here](./terminal-example)
+
 ### Some of the often used terminal commands 
 
 
 * `ls` - to list the content of the folder
-* `mkdir` - to make new folder
+* `mkdir` - to make a new folder
 * `cp` - to copy files
 * `rm` - to delete files and folders
-* ...
 
+On unix based systems like Linux or MacOS every command line option help can be checked by using `man` command followed by the name of the command. 
```

Comment:
On unix based systems like Linux or MacOS every command line option help can be checked by using the `man` command followed by the name of the command.